### ISSUE TITLE
Reminder -- Specify the key range to search for prefix - new feature request

### ISSUE THREAD

**Issue by hamnaz** [JeanLucPons/VanitySearch#80]
Specify the range of keys to search. For example, if you want to search in a specific range of private keys you can specify it in hex or decimal in the format START:END with a specified stride/increment size.

+1 need this function too , with keyspace start: end , with a specified stride/increment size.
like stride 10000:10000, with user define in stride and checkkeys
example user define
check keys 10000
stride keys 10000
this func make Vanitysearch more attractive and chances of probability build up

**Comment by futuremoneybtc** [comment]
maybe he is already done it but not published here.

**Comment by hamnaz** [comment]
yes maybe, because he is high tech developer, and these updates and work for him is just in few min, i see he were online daily , from last 3 month he apear online only once in 10 days, maybe busy in other project, let see when he spear some time for community, and give them their required updates and developments, for find puzzle solutions, and learning

**Comment by futuremoneybtc** [comment]
Do you have a max prefix of puzzle 64 like this?  
**16jY7qLJn**xb7CHZyqBP8qca9d**5**1gAjyXQN
**16jY7qLJn**NtLkE51fgNsMqYAZ**5**i6LqURJm - **F262E8D54F236F43**

**Comment by futuremoneybtc** [comment]
Or from the back like this?  
**16**jY7qLJnxb7CHZyqBP8qca9d51gA**jyXQN**
**16**fphXLUnzwsBZwW4MpayMyZwFw8D**jyXQN** - **F9E6DAFA40B34B04**

**Comment by MrBForcer** [comment]
> Or from the back like this? **16**jY7qLJnxb7CHZyqBP8qca9d51gA**jyXQN** **16**fphXLUnzwsBZwW4MpayMyZwFw8D**jyXQN** - **F9E6DAFA40B34B04**

**16jY7qL**HKTofWA2DYS9ZaWyvZ5qnjfS**XQN** - daa7b7bdf277a7b7
